src/platform_impl/macos/mod.rs

```diff
@@ -988,6 +1009,63 @@ extern "C" fn fire_menu_item_click(this: &Object, _: Sel, _item: id) {
         let ptr: usize = *this.get_ivar(BLOCK_PTR);
         let item = ptr as *mut &mut MenuChild;
 
+        if let PredfinedMenuItemType::About(about_meta) = &(*item).predefined_item_type {
+            match about_meta {
+                Some(about_meta) => {
+                    unsafe fn mkstr(s: &str) -> id {
+                        NSString::alloc(nil).init_str(s)
+                    }
+
+                    let mut keys: Vec<id> = Default::default();
+                    let mut objects: Vec<id> = Default::default();
+
+                    if let Some(name) = &about_meta.name {
+                        keys.push(NSAboutPanelOptionApplicationName);
+                        objects.push(mkstr(&name));
+                    }
+
+                    if let Some(version) = &about_meta.version {
+                        keys.push(NSAboutPanelOptionApplicationVersion);
+                        objects.push(mkstr(&version));
+                    }
+
+                    if let Some(short_version) = &about_meta.short_version {
+                        keys.push(NSAboutPanelOptionVersion);
+                        objects.push(mkstr(&short_version));
+                    }
+
+                    if let Some(copyright) = &about_meta.copyright {
+                        keys.push(NSString::alloc(nil).init_str("Copyright"));
```

Comment:
Is there a clean way to make this a global constant? I wasn't able to do it using `static` since `*mut Object` is not thread-safe.

Reply:
that's fine, you can just make the `Copyright` string a constant and use it
```suggestion
                        keys.push(mkstr(NSAboutPanelOptionCopyright));
```